### PR TITLE
feature: Update organizations page UX

### PR DESCRIPTION
**Pull request by pauloribeiro-codacy**
Updates the documentation with the UI/UX changes resulting from moving the organizations page to React. See [CY-3001](https://codacy.atlassian.net/browse/CY-3001).

### CODE REVIEW

**Review by inesbpinto — APPROVED** [review]

File: docs/organizations/what-are-synced-organizations.md [hunk 1]
```diff
@@ -12,13 +12,13 @@ To add a new organization to Codacy, select **Add organization** on the navigati
 
 This opens the list of organizations on your Git providers. The organizations with the same name as your Git provider username contain your personal repositories.
 
--   To add a new organization to Codacy, click the button **Add** for that organization.
--   To add organizations from a Git provider that is not yet listed on this page, click **Add provider** and give the [necessary permissions](../getting-started/which-permissions-does-codacy-need-from-my-account.md) for Codacy to sync with the new Git provider and display your organizations.
+-   To add a new organization to Codacy, click the link **Add** for that organization.
+-   To add organizations from a Git provider not yet listed on this page, click **Add provider** and give the [necessary permissions](../getting-started/which-permissions-does-codacy-need-from-my-account.md) for Codacy to sync with the new Git provider and display your organizations.
 
 ![Adding an organization](images/organization-add.png)
```
[the remark this answers is not in the document]
Comment (inesgomas) [reply docs/organizations/what-are-synced-organizations.md:18]: You can just delete you personal org to see the Add button

**Comment by pauloribeiro-codacy** [comment]
Here are previews of the pages that are updated on this pull request:

- https://github.com/codacy/docs/blob/feature/update-organizations-page-CY-3216/docs/organizations/what-are-synced-organizations.md#adding-an-organization
- https://github.com/codacy/docs/blob/feature/update-organizations-page-CY-3216/docs/faq/general/why-cant-i-see-my-organization.md

File: docs/organizations/what-are-synced-organizations.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (pauloribeiro-codacy) [reply docs/organizations/what-are-synced-organizations.md:19]: Well caught @inesgomas! :female_detective: I updated the screenshot and I also mention that the user can join an organization if it already exists on Codacy. :+1:

**Review by inesgomas — COMMENTED** [review]
I don't think the image organization-join.png is behind used 👀

**Comment by alerizzo** [comment]
One thing, not critical nor important, but an aesthetic issue only. It would be great if we take the screenshots without the accessible-scrollbars enabled in the browser:

image

Or probably is a matter of OS / browser version?? This is how I see it, for example:

image

**Comment by pauloribeiro-codacy** [comment]
> I don't think the image organization-join.png is behind used

@inesgomas, it's used on this other file:

https://github.com/codacy/docs/blob/feature/update-organizations-page-CY-3216/docs/organizations/authors-vs-members.md

Since I didn't update anything on this Markdown file it doesn't appear in the changes for this pull request.

**Comment by pauloribeiro-codacy** [comment]
> One thing, not critical nor important, but an aesthetic issue only. It would be great if we take the screenshots without the accessible-scrollbars enabled in the browser:
> 
> [...]
> 
> Or probably is a matter of OS / browser version?? This is how I see it, for example:
> 
> [...]

@alerizzo, I do not have any accessibility settings enabled on the browser I use to collect the screenshots. I believe that the scrollbar is defined by the browser and operating system, but I'm not sure!

Here are example screenshots from (left to right) Google Chrome, Mozilla Firefox, and Microsoft Edge running on Linux Mint:

image

Here are the same browsers but running on Windows:

image

I believe that any user not using macOS will see the scrollbars like this and not like in your example. :thinking:

**Review by inesgomas — DISMISSED** [review]

**Review by inesgomas — APPROVED** [review]

**Pull request merged**